dbmanager.py

```diff
@@ -31,23 +40,38 @@ def exec_sql(connection, sql):
     :returns None if the connection or sql was None,
              a cursor containing the results of the statement otherwise
     :raises A MySqlError (likely due to an error in the SQL statement syntax)
+    
+    example: 
+        
+        from testdb.py
+        import dbmanager
+        connection = dbmanager.init_connection('MySQLusername', 'MySQLpassword')
+        sql = 'SELECT * FROM locations LIMIT 10;'
+        result = dbmanager.exec_sql(connection, sql)
+        print(result)
     """
     cursor = None
     if sql and connection:
```

Comment:
Are sql and connection guaranteed by this point? If not, else? Otherwise, why keep the if statement?